src/ngraph/runtime/cpu/mkldnn_emitter.hpp

```diff
@@ -198,7 +205,24 @@ namespace ngraph
                     }
                     else if (std::is_same<OP, ngraph::op::QuantizedConvolution>())
```

Comment:
You can use 1 if statement to check for both OP types? and pass `OP` to get_scale()?

Reply:
I tried that. But for some reason the templates try to use all possible values of OP and hence it crashes. Maybe a template within template thing.

Reply:
ok. no worries.